docs/components/ComponentsList/index.js

```diff
@@ -47,11 +46,11 @@ export function ComponentsList({ onClick, ...props }) {
                 <Link href={`/components/${slugify(item)}`} passHref>
                   <S.Item>
                     {item}
-                    {newItems.includes(slugify(item)) ?
-                      <Badge size="sm" variant="primary">new</Badge>
-                    :
-                      ''
-                    }
+                    {newItems.includes(slugify(item)) ? (
+                      <Badge size="sm" variant="primary">
+                        new
+                      </Badge>
+                    ) : null}
```

Comment:
I'd rather have :
`
{newItems.includes(slugify(item)) && (
                      <Badge size="sm" variant="primary">
                        new
                      </Badge>
                    )}`
since we usually don't use ternary in return